docs/appendices/release-notes/unreleased.rst

```diff
@@ -55,6 +55,9 @@ Changes
 
 - Added support for the :ref:`CREATE TABLE AS <ref-create-table-as>` statement.
 
+- Added an empty ``pg_tablespace`` table in the ``pg_catalog`` schema for
```

Comment:
should be on top of the list or?

Reply:
if there are no other considerations, yes, I believe that the list should be ordered by date descending. however, there are two reasons I see to deviate from this default:

- when an item ought to be highlighted (e.g., it's important), it can be moved to the top
- if there is a logical order for some changes (e.g., one follows on from another), then I think ordering them logically is okay

Reply:
Changed it to top, though I don't really understand why we're often so strict about this.

We usually group the release notes by topic and importance shortly before a release anyways, so to me the main point here is to make resolving conflicts (if they happen) easier, and that is the same for adding on top or adding at the bottom.

Keep in mind that the release notes are for users, and for them they're reading the changes as a diff between one version to another. The temporal order in which a change was merged is mostly irrelevant for them. They don't care whether table `a` was added before table `b` within the same release.

Reply:
Hm I think you're right. 👍